package/skelcd-control-openSUSE.changes

```diff
@@ -1,9 +1,15 @@
 -------------------------------------------------------------------
+Fri Aug 26 12:45:20 UTC 2016 - igonzalezsosa@suse.com
+
+-  Move the YaST self update step earlier in the workflow to avoid
+   bugs caused by restarting YaST (boo#995771)
+- 42.2.3
```

Comment:
Yes: *Version is not bumped because the last submit request was rejected (due to missing bugnumber)* :)